package/linux/scripts/upload-debug-symbols

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+#
+# upload-debug-symbols
+#
+# Copyright (C) 2025 by Posit Software, PBC
+#
+# Unless you have received this program directly from Posit Software pursuant
+# to the terms of a commercial license agreement with Posit Software, then
+# this program is licensed to you under the terms of version 3 of the
+# GNU Affero General Public License. This program is distributed WITHOUT
+# ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+# MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+# AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+#
+
+# TODO: Find a more appropriate bucket?
+AWS_BUCKET="s3://rstudio-buildtools"
```

Comment:
Let's make a bucket in our sub-account specifically for this purpose. Maybe s3://release-symbols?

Reply:
I went with `s3://rstudio-dbgsym`, owned by the IDE team sub-account.